jsonrpc/server.go

```diff
@@ -310,8 +311,10 @@ func buildArguments(params, handler any, configuredParams []Parameter) ([]reflec
 	case reflect.Slice:
 		paramsList := params.([]any)
 
-		if len(paramsList) != handlerType.NumIn() {
-			return nil, errors.New("missing param in list")
```

Comment:
I dont think we need to handle this in two different conditionals.
Let's just update the error message to be generic like; "missing/unexpected params in list" and we are good

Reply:
Done